dynamodb-stream-reader/src/test/java/no/unit/nva/elasticsearch/ElasticSearchIndexDocumentTest.java

```diff
@@ -0,0 +1,61 @@
+package no.unit.nva.elasticsearch;
+
+import elasticsearch.ElasticSearchIndexDocument;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ElasticSearchIndexDocumentTest {
+
+    private static final String INDEX_NAME = "someIndex";
+    private static final String SERVICE_URL = "http://localhost/";
+    private static final String INTERNAL_IDENTIFIER = "id";
+    public static final String SAMPLE_CONTRIBUTOR = "Doe, John";
+    public static final String SAMPLE_CONTRIBUTOR2 = "Doe, Jane";
+    public static final String SAMPLE_CONTRIBUTOR3 = "Andrè, Noen";
+    public static final String SAMPLE_DATE = "2020-08-26";
+    public static final String SAMPLE_RESOURCETYPE = "sampleResourceType";
+    public static final String SAMPLE_TITLE = "This Is A Sample Title";
+
+    @Test
+    @DisplayName("Testing ElasticSearchIndexDocument constructor and initializing")
+    public void testElasticSearchIndexDocumentConstructor() {
```

Comment:
fix naming. You do not need to start with the word test. 
suggestion: constrcuctorCreatesDocumentThatIncludesIndexServiceUrlAndInternalIdentifier

Reply:
We need to document test naming strategy